### DISCUSSION TITLE
How to calculate ptm scores of an existing protein 3D complex structure ?

### DISCUSSION THREAD

**Discussion by gciaberi** [chaidiscovery/chai-lab#232] [Announcements]
I would be interested in calculating the ptm/iptm/chain_ptm etc scores for a 3D protein complex structure that I have. 

I could of course run Chai-1r with a very large number of restraints for the interacting interface residues so that the generated model matches almost exactly with my structure, but I was wondering if it's possible to skip the whole modelling part and just calculate the scores ? Would this be doable, and if yes, what would be the steps to do it ?

Thanks !

**Comment by arogozhnikov** [comment]
Hi @gciaberi 

confidence estimate (like pTM) were trained only on model predictions and they *require* model predictions besides just atom positions. So you can't skip modelling.

**Comment by gciaberi** [comment]
@arogozhnikov Ok I see, thanks for the information. I have tested the approach of using many restraints, and after some testing and a total of 17 restraints, I managed to get models with both ptm and iptm scores > 0.8 which are almost identical to the known complex structure.
